src/ui/layout/tests/Layout.test.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+
+import { Layout } from "..";
+
+
+describe('<Layout>', () => {
+    it("shows children", () => {
+        const child = <div data-testid="child">Test</div>;
+        render(<Layout>{child}</Layout>)
+        expect(screen.getByTestId("child")).toBeDefined();
+    })
+
+    it("style pass correctly", () => {
+        const child = <div data-testid="child">Test</div>;
```

Comment:
remove variable and test id.
Change test title - applies className - without correctly

Reply:
done